mm2src/coins/eth/for_tests.rs

```diff
@@ -47,6 +47,7 @@ pub(crate) fn eth_coin_from_keypair(
         EthCoinType::Eth => "ETH".to_string(),
         EthCoinType::Erc20 { .. } => "JST".to_string(),
         EthCoinType::Nft { ref platform } => platform.to_string(),
+        EthCoinType::Trx => "TRX".to_string(),
```

Comment:
We can implement `Display` trait for `EthCoinType` which will allow us to avoid this kind of pattern matching and print it with the the debug mode (`{:?}`) in the error messages.